docs/tutorials.state_distinguishability.rst

```diff
@@ -112,29 +112,27 @@ can optimally distinguish the state he is given perfectly, i.e. with probability
 
 Using :code:`toqito`, we can calculate this probability directly as follows:
 
-.. code-block:: python
-
-    >>> import numpy as np
-    >>> from toqito.states import basis
-    >>> from toqito.state_opt import state_distinguishability
-    >>> 
-    >>> # Define the standard basis |0> and |1>
-    >>> e_0, e_1 = basis(2, 0), basis(2, 1)
-    >>>
-    >>> # Define the corresponding density matrices of |0> and |1> 
-    >>> # given as |0><0| and |1><1|, respectively.
-    >>> e_00 = e_0 @ e_0.conj().T
-    >>> e_11 = e_1 @ e_1.conj().T
-    >>>
-    >>> # Define a list of states and a corresponding list of 
-    >>> # probabilities with which those states are selected.
-    >>> states = [e_00, e_11] 
-    >>> probs = [1/2, 1/2]
-    >>>
-    >>> # Calculate the probability with which Bob can 
-    >>> # distinguish the state he is provided.
-    >>> np.around(state_distinguishability(states, probs)[0], decimals=2)
-    np.float64(1.0)
+.. jupyter-execute::
+
+     import numpy as np
+     from toqito.states import basis
+     from toqito.state_opt import state_distinguishability
+     
+     # Define the standard basis |0> and |1>
+     e_0, e_1 = basis(2, 0), basis(2, 1)
+    
+     # Define the corresponding density matrices of |0> and |1> 
+     # given as |0><0| and |1><1|, respectively.
+     e_00 = e_0 @ e_0.conj().T
+     e_11 = e_1 @ e_1.conj().T
```

Comment:
[nit]: Hmm, I seem to think the `e_00` denotes more `np.kron(e_0, e_0)` and that the standard basis matrix, `E_0` would be a better name for these variables:

```suggestion
     E_0 = e_0 @ e_0.conj().T
     E_1 = e_1 @ e_1.conj().T
```